Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/TimeStamper.kt

```diff
@@ -0,0 +1,12 @@
+package de.rki.coronawarnapp.util
+
+import org.joda.time.Instant
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class TimeStamper @Inject constructor() {
+
+    val nowUTC: Instant
```

Comment:
not consequently used. I'd prefer Instant.now() instead of this helper.

Reply:
I agree, we should probably also replace all other calls to Instant.now().

Reply:
I think we do need this class so that testing is easier and also we can easily fake a timestamp for the whole project

Reply:
It's not "yet" consequently used when refactoring please make use of this class where possible. It allows us to inject mock time sources in testing and "timetravel" to test edge cases.

Reply:
IMHO this can be improved by switching to the ThreeTen time library entirely injecting a Clock that - which in turn can be moved in time - instead of using helper functions to solely determine the current time. Anyway, this is a bigger discussion for another time.